packages/machine/src/protocol/install.ts

```diff
@@ -52,6 +53,17 @@ export const INSTALL_PROTOCOL = {
     // Wrap the signature into a message to be sent
     prepareToSendSignature,
 
+    function prepareToSendSignature(
```

Comment:
what's this step for?

Reply:
Swaps `fromAddress` and `toAddress` before sending it to the counterparty. The only reason the function is called `prepareToSendSignature` is that I copy and pasted that function into this location and was moving too quickly to care to delete the function name, so it makes no sense out of context sorry.